FlaskApp/app/views.py

```diff
@@ -5,41 +5,82 @@
 # dude these schema things should not be in the view.. this basically means
 # everytime when you refresh the page you have to reconnect the server??
 # I believe this should be in init file and the you shoudl find a way to maintain through model.
+from datetime import datetime, timedelta
 from flask import render_template, redirect, url_for, request
 from app.schema.tweets import Tweets_sche
 import pymongo
 from pymongo import MongoClient
 from pymongo.errors import ConnectionFailure
-# this file should works as part of controllers for routes
-# if we have more actions later we can have a folder for controllers later but now lets keep it here.
-# all routes should go here
+import logging
+from logging import Formatter, FileHandler
+
+#Setup the logger
+LOGGER = logging.getLogger('streamer_logger')
+file_handler = FileHandler('streamer.log')
+handler = logging.StreamHandler()
+file_handler.setFormatter(Formatter(
+        '%(thread)d %(asctime)s %(levelname)s: %(message)s '
+        '[in %(pathname)s:%(lineno)d]'
+))
+handler.setFormatter(Formatter(
+        '%(thread)d %(asctime)s %(levelname)s: %(message)s '
+        '[in %(pathname)s:%(lineno)d]'
+))
+LOGGER.addHandler(file_handler)
+LOGGER.addHandler(handler)
+LOGGER.setLevel(logging.DEBUG)
+
 @app.route("/")
 def index():
+    return render_template('index.html')
 
-	#----DB INIT BEGIN---------------------------------------------
-
-	MONGO_HOST = 'localhost'
-	MONGO_PORT = 27017
 
-	#Try to connect to MongoDB
-	try:
-		client = MongoClient(MONGO_HOST, MONGO_PORT)
-		db = client.tinyjumbo
-		tweet_collection = db.google
-		print "Sucees connect to DB"
+@app.route("/barchart")
+def bar():
+    #----DB INIT BEGIN---------------------------------------------
 
-	except ConnectionFailure:
-		LOGGER.error('Could not connect to MongoDB, aborting Flask app...')
-		sys.exit(-1)
-	#----DB INIT END---------------------------------------------
+    MONGO_HOST = '162.243.122.37'
```

Comment:
do it via set ENV in bashrc or bash_profle 
vim ~/.bashrc

MONGO_HOST='xxxx'

source ~/.bashrc

in your py file
os. environ['MONGO_HOST'] is the varibale you defined. so that you can call it here.
in your own machine you can define it as localhost. 

same use case for path variables